ui/src/pages/services/index.tsx

```diff
@@ -77,129 +88,101 @@ export default function ServicesPage() {
-                  <Icon name="circle-house" />
+              <li className="flex flex--start margin-top-small">
+                <div className="margin-right-small card card--icon card--elevation-2">
+                  <Icon name="circle-lock" />
                 </div>
-                Give everyone the chance to explore your delicious dishes. PDF menus are not responsive, which means it
-                takes a whole lot of zooming and scrolling to navigate them. Our QR menus are web pages, so they scale
-                perfectly on almost any mobile device.
+                Tie in your preferred content management system (CMS) to seamlessly integrate new blog posts, photos, or
+                portfolio pieces into your site. We're all about convenience, and we strive to make content management
+                as simple as possible for all of our clients.
               </li>
-              <li>
-                <div className="card card--icon card--elevation-2">
-                  <Icon name="smiley-face" />
+              <li className="flex flex--start margin-top-small">
+                <div className="margin-right-small card card--icon card--elevation-2">
+                  <Icon name="thunder-bolt" />
                 </div>
-                Efficiently add, remove, and edit your offerings. Updating paper menus can be costly and time-consuming,
-                but websites are built to be changed. When you cook up new dishes or switch your seasonal offerings, QR
-                code menus make the transition seamless.
+                Reach an international audience. Our team’s experience with multilingual compatibility ensures that your
+                site will look and feel perfect for a diverse group of people, not just English speakers.
               </li>
             </ul>
-            <div>IMAGE</div>
           </div>
         </div>
       </Section>
-      <Section>
-        <div className="grid">
-          <div>
-            <h3 className="margin-top-0">Digital Menus</h3>
-            <p>
-              If you’re a restaurant owner looking to stay on the cutting edge of the customer experience, building a
-              smooth, responsive Digital menu is crucial to the new digital age of ordering. The COVID-19 pandemic has
-              caused thousands of businesses to switch to Digital menus, and it looks like they’re here to stay. Our
-              custom Digital menu creation service, Code My Menu, offers flexible, personalized solutions that are
-              compatible with almost any mobile device.
-            </p>
-            <p>
-              See our full stack web development work in action for{' '}
-              <a target="_blank" rel="nofollow noopener" href="https://www.leveluptutorials.com/?ref=purefunc">
-                Level Up Tutorials.
-              </a>{' '}
+        <div className="section section--banner section--padding">
```

Comment:
Let's try to use the `<Section>` component for sections instead of just adding section classes to keep the code DRY and give us the ability to pass props